TF2 Rich Presence/locales/Spanish.json

```diff
@@ -1,15 +1,15 @@
 {
     "name_localized": "Español",
     "code": "es",
-    "credits": "",
+    "credits": "DznDani",
     "notes": "",
     "4021625669": "(Si tiene más de una versión desactualizada, es posible que haya habido más cambios y correcciones que ésta.)",
-    "3336272700": "1. Haz clic con el botón derecho del ratón en Team Fortress 2 en tu biblioteca de Steam",
-    "3034581868": "2. Propiedades abiertas (muy abajo)",
-    "2781874761": "3. Haga clic en \"Establecer opciones de lanzamiento...\"",
-    "2276072520": "4. Añadir {0}",
-    "5268450630": "5. Aceptar y Cerrar",
-    "5334641500": "6. Reiniciar TF2",
+    "3336272700": "1. Haz click derecho en Team Fortress 2 en tu biblioteca de Steam",
+    "3034581868": "2. Abre propiedades",
+    "2781874761": "3. Haz click en \"Parametros de Lanzamiento...\"",
```

Comment:
"Clic" is actually the correct version. Here you have a [Tweet clarifying that from RAEConsultas](https://twitter.com/raeinforma/status/1221741614177177600) and this is the [respective entry in the RAE dictionary](https://www.rae.es/dpd/clic).